core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -223,11 +225,11 @@ object ConsumerGroupCommand {
                 .format(group, topicAndPartition))
           }
         }
-        else if (offsetAndMetadata.error == ErrorMapping.NoError)
+        else if (offsetAndMetadata.error == Errors.NONE.code)
```

Comment:
We have a pattern in the code of passing error codes around as shorts. Would it make sense to pass instances of `Errors` instead? (I'm not saying this has to be done in this already large patch)